systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -148,6 +149,7 @@ protected void installClusterOperator(String namespace) throws Exception {
      * @param resources list of path to yaml files with resources specifications
      */
     protected void prepareEnvForOperator(String clientNamespace, List<String> namespaces, String... resources) {
+        assumeTrue(!Environment.isHelmInstall() && !Environment.isOlmInstall());
```

Comment:
Hmmm, don't we use `asserThat(condition, is(true));` on everything?

Reply:
assume != assert